deployment.rb

```diff
@@ -107,7 +108,7 @@ def load_configuration
     'assets_s3_bucket'            => 'cdo-v3-assets',
     'assets_s3_directory'         => rack_env == :production ? 'assets' : "assets_#{rack_env}",
     'sources_s3_bucket'           => 'cdo-v3-sources',
-    'sources_s3_directory'        => rack_env == :production ? 'sources' : "sources_#{rack_env}",
+    'sources_s3_directory'        => sources_s3_dir(rack_env),
```

Comment:
Looks like you're passing an argument here, but not accepting one in the helper written below.

Reply:
Fixed!